src/main/java/br/com/carrental/service/UserServiceImpl.java

```diff
@@ -0,0 +1,103 @@
+package br.com.carrental.service;
+
+import br.com.carrental.service.dto.UserDTO;
+import br.com.carrental.service.exception.UserAlreadyExistsException;
+import br.com.carrental.service.exception.UserNotFoundException;
+import br.com.carrental.model.User;
+import br.com.carrental.service.repository.UserRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class UserServiceImpl {
+    @Autowired
+    private UserRepository repository;
+
+    private final Logger LOGGER = LoggerFactory.getLogger(UserServiceImpl.class);
+
+    public ResponseEntity getAllUsers() {
+        List<User> list = repository.findAll();
+        List<UserDTO> listDTO = new ArrayList<>();
+        for (User user : list) {
+            listDTO.add(new UserDTO(user.getIdDocument(),
+                    user.getName(),
+                    user.getEmail(),
+                    user.getAddress(),
+                    new SimpleDateFormat("dd/MM/yyyy")
+                            .format(user.getBirthDate())));
+        }
+        LOGGER.info("m=getAllUsers: GET all users success");
```

Comment:
Em que cenário vc logaria algo diferente de "GET all users success"? Se vc for criar um log apenas informativo (sem criar logs de início e fim de processamento para calcular o tempo de execução, por exemplo), vc poderia convencionar deixá-los no início do método.